src/main/java/io/github/syst3ms/skriptparser/expressions/ExprStringCharAt.java

```diff
@@ -0,0 +1,53 @@
+package io.github.syst3ms.skriptparser.expressions;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import io.github.syst3ms.skriptparser.Parser;
+import io.github.syst3ms.skriptparser.lang.Expression;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+import io.github.syst3ms.skriptparser.util.DoubleOptional;
+
+/**
+ * The character at a position of a string. The first character is at the position 0.
```

Comment:
```suggestion
 * The character at a given position in a string. Note that indices in Skript start at 1.
```
Truly a nitpick, but that indices change is rather important.